{{Introduction to Python Basics: Variables and Data types}}/.md

```diff
@@ -0,0 +1,180 @@
+
+Python download: https://www.python.org/downloads/
+
+
+
+The first topic that we will be going over are Data Types. 
+
+Data Types
+
+Data Types are described as specific classifications or categories for data items. In Python, the most common 3 Data Types are: Integers, Floating Points and Strings. These are the 3 that we will be going over today. 
+
+a.Integers:
+
+Integers in Python are characterized as positive or negative whole numbers like -1,0,1,2,3. Think of integers as real numbers, no decimal points just whole numbers. Integers can not be words. Although the word ‘seven’ represents the actual number 7 this is not considered an Integers. Remember integers are only whole numbers. 
+
+Ex 1.1:
+
+	integer = -1
+	integer = 1
+	integer = 0
+
+These are examples that demonstrates integer values. 
+
+Ex 1.2:
+
+	integer = seven
+
+
+This example represents what I previously mentioned. Although the word seven represents the number 7 Python sees it as a word not an actual value.
+
+
+b.Floating points:
+
+Floating points are similar to integers except these numbers contain decimal points. For example: 42.0, 3.1415, and -89.12 are all examples of Floating points. 
+
+Ex 1.3:
+
+	floating = 42.0
+	floating = 3.1415
+	floating = -89.12
+
+Here we can see examples of floating point values and how they all contain decimal points. Also notice that floating points can be negative as well.
+
+c.Strings:
+
+The final Data Types that we will be talking about are strings. Strings are described as a sequence of characters enclosed by quotation marks. These characters can be word or even numbers. The quotation marks can either be single quotes( ‘’) or double quotes (“”) it doesn't matter which you decide to choose however you must be uniform with whichever you choose. 
+
+Ex 1.4: 
+
+	string = 'Welcome to this blog!'
+
+This is the correct way to use single quotes. You can notice that the string starts with a single quote and ends with a single quote which is what we want to do.
+
+Ex 1.5: 
+
+	string = 'Welcome to this blog!"
+
+Python will not like this string because we do not end with the same type of quote that we started with, we aren't being uniform. The string begins with a single quote and ends with a double quote. If we start with a single quote we must end with a single quote. 
+
+Ex 1.6:
+
+	string = '42'
+
+This is another example of a string. Although we would think that ‘42’ is an integer that is not the case for this particular example. Since the number 42 is in quotation marks it is actually considered a string. Any numbers that are enclosed in quotation marks are considered string. 
```

Comment:
nit
```suggestion
This is another example of a string. Although we would think that ‘42’ is an integer that is not the case for this particular example. Since the number 42 is in quotation marks it is actually considered a string. Any numbers that are enclosed in quotation marks are considered a string. 
```